index.bs

```diff
@@ -2547,17 +2547,17 @@ Proposed Changes</h4>
 	<dfn lt="proposed change">proposed changes</dfn>.
 
 <h4 id="contributor-license">
-Contributor License Grants</h4>
+License Grants from Non-Participants</h4>
 
-	When a contributor who is not already obligated under the Patent Policy
+	When a party who is not already obligated under the Patent Policy
```

Comment:
s/a party/somebody/ ?

Reply:
No, because the Process is currently vague on whether "contributor" refers to an individual or the company they work for, and this needs to stay ambiguous.